jest.polyfills.js

```diff
@@ -0,0 +1,31 @@
+// jest.polyfills.js
+/**
+ * @note The block below contains polyfills for Node.js globals
+ * required for Jest to function when running JSDOM tests.
+ * These HAVE to be require's and HAVE to be in this exact
+ * order, since "undici" depends on the "TextEncoder" global API.
+ *
+ * Consider migrating to a more modern test runner if
+ * you don't want to deal with this.
```

Comment:
Huh? xD What does this mean? I guess its copied over, but do they mean more modern then Jest, or newer version of Jest?

Reply:
I think it's because we use jsdom in jest test env, we can pick a different test environment.

Reply:
Up to you, I'll go with what you think is best solution, I was just curious ;)

Reply:
This polyfills file  is taken from the official `msw` migration guide, so let's use it instead of replacing to a different test env :)